algo/src/main/kotlin/com/hartwig/actin/algo/evaluation/treatment/HasHadTreatmentWithCategoryOfTypesRecently.kt

```diff
@@ -37,26 +37,22 @@ class HasHadTreatmentWithCategoryOfTypesRecently(
             )
         }.fold(TreatmentAssessment()) { acc, element -> acc.combineWith(element) }
 
-        val typesList = concatItemsWithOr(types)
+        val typesList = types?.let { "${concatItemsWithOr(types)} " } ?: ""
```

Comment:
Since this is always displayed with category, it could be:
```suggestion
        val typesAndCategoryString = listOfNotNull(types?.let(::concatItemsWithOr), category.display()).joinToString(" ")
```
so you don't have to concatenate with category each time below.